### DISCUSSION TITLE
Local Lunar Eclipse

### DISCUSSION THREAD

**Discussion by bjarkeandersen** [cosinekitty/astronomy#255] [Q&A] [answered]
I see there is a SearchLocalSolarEclipse function for solar eclipses, but for lunar eclipses there is no local function. How can I calculate information about the next lunar eclipse seen from a specific point on Earth?

**Comment by cosinekitty** [comment, marked as answer]
Hi @bjarkeandersen. My thinking is, a solar eclipse is highly dependent on the viewer's location, because the Moon's shadow either falls on you or it doesn't, depending on where you are. However, a lunar eclipse looks the same to everyone around the world, because they are all watching the Earth's shadow fall upon the Moon.

The main effect of geographic location is whether the Moon is visible where you are at the time the eclipse happens. Once you have calculated the start, peak, and end of a lunar eclipse, there are separate functions in Astronomy Engine to determine where the Moon appears in the sky for a given observer. Then you can determine whether the Moon is above or below the horizon at the times of interest.

Does this help?

**Reply by bjarkeandersen** [reply]
I was checking at timeanddate and thought that the next lunar eclipse on november 8th was only a penumbral eclipse where I live. But now I see that the reason is that the moon is below the horizon untill the very end of the total eclipse. That makes more sense - i.e. using daylight information and moon elevation to determine how much of the lunar eclipse is actually visible from a coordinate on Earth. Thanks for a very fast answer!
